src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,117 @@
+        System.out.println(System.lineSeparator() + "Get all drivers from DB:");
+        drivers.forEach(System.out::println);
+
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        CarService carService =
+                (CarService) injector.getInstance(CarService.class);
+
+        Car mercedes = new Car();
+        Manufacturer mercedesBenzManufacturer =
+                manufacturerService.create(new Manufacturer("Mercedes-Benz", "Germany"));
+        mercedes.setManufacturer(mercedesBenzManufacturer);
+        mercedes.setModel("C 300");
+        mercedes = carService.create(mercedes);
+
+        Car bentley = new Car();
+        Manufacturer bentleyManufacturer =
+                manufacturerService.create(new Manufacturer("Bentley", "England"));
+        bentley.setManufacturer(bentleyManufacturer);
+        bentley.setModel("Continental GT Coupe V8");
+        bentley = carService.create(bentley);
+
+        Car lexus = new Car();
+        Manufacturer lexusManufacturer =
+                manufacturerService.create(new Manufacturer("Lexus", "Japan"));
+        lexus.setManufacturer(lexusManufacturer);
+        lexus.setModel("Lexus LC Convertible");
+        lexus = carService.create(lexus);
+
+        Car ford = new Car();
+        Manufacturer fordManufacturer =
+                manufacturerService.create(new Manufacturer("Ford", "USA"));
+        ford.setManufacturer(fordManufacturer);
+        ford.setModel("F-150");
+        ford = carService.create(ford);
+
+        List<Manufacturer> manufacturers = manufacturerService.getAll();
+        System.out.println(System.lineSeparator() + "Get all manufacturers from DB:");
+        manufacturers.forEach(System.out::println);
+
+        carService.addDriverToCar(drivers.get(1), mercedes);
+        carService.addDriverToCar(drivers.get(2), mercedes);
+        carService.addDriverToCar(drivers.get(0), bentley);
+        carService.addDriverToCar(drivers.get(2), bentley);
+        carService.addDriverToCar(drivers.get(0), lexus);
+        carService.addDriverToCar(drivers.get(1), lexus);
+        carService.addDriverToCar(drivers.get(3), ford);
+
+        Long randomCarId = random.longs(1, 5)
+                .limit(1)
+                .findFirst()
+                .getAsLong();
+        Car randomCarFromDb = carService.get(randomCarId);
+        System.out.println(System.lineSeparator()
+                + "Get random car from DB: " + System.lineSeparator()
+                + randomCarFromDb);
+
+        randomCarFromDb.getManufacturer().setName("Roman Yuskevych");
+        randomCarFromDb.getManufacturer().setCountry("Ukraine");
+        randomCarFromDb.setModel("BestModelEver");
+        manufacturerService.update(randomCarFromDb.getManufacturer());
+        carService.update(randomCarFromDb);
+        while (randomCarFromDb.getDrivers().size() > 0) {
```

Comment:
Why do we use loop here for 1 iteration?
```suggestion
        if (randomCarFromDb.getDrivers().size() > 0) {
```